sycl/test-e2e/AddressSanitizer/bad-free/bad-free-minus1.cpp

```diff
@@ -1,4 +1,5 @@
-// REQUIRES: linux, cpu
+// UNSUPPORTED: cuda || hip
+// REQUIRES: linux
```

Comment:
Looks like that is the same for all the test in the folder. If so, `local.lit.cfg` is a better place to have this. The syntax is different, but one can look at other local config files to copy-paste from.

Reply:
Thanks. I leaves "REQUIRES" in the test, some tests will force to use gpu device in the future.